frontend/projects/wc/src/app/showcase/showcase.component.html

```diff
@@ -1,15 +1,41 @@
+<div class="title">
+  <ui5-title  level="H3">{{title}}</ui5-title>
```

Comment:
we should consider to use prettier so all formatting is done automatically
```suggestion
  <ui5-title level="H3">{{title}}</ui5-title>
```